problems/sortedClassesBySize.js

```diff
@@ -11,7 +11,14 @@
 * returns: ["B", "C", "A"]
 */
 
-function sortClassesBySize() {
+function sortClassesBySize(classes) {
+    classes.sort((subject1, subject2) => {
+        return subject1 - subject2
+    }).map((subject) => {
+        return subject.subject
+    })
```

Comment:
very close, you forget two things. To return a value to the function from the chaining of two array method, and to access the key `studentCount` in line 16